OsmAnd/src/net/osmand/plus/exploreplaces/ExplorePlacesFragment.kt

```diff
@@ -390,6 +478,8 @@ class ExplorePlacesFragment : BaseOsmAndFragment(), NearbyItemClickListener,
 		private const val REFRESH_UI_ID = EXPLORE_PLACES_UPDATE + 1
 		private const val POI_UI_FILTER_ID = "poi_ui_filter_id"
 
+		private const val LANDSCAPE_SLIDE_ANIMATION_TIME = 500L
```

Comment:
in context menu we use system animation time, look inside [slide_in_bottom](https://github.com/osmandapp/Osmand/blob/a8585ea4fc3aea1aa437d67e27dd8e52ab516c4e/OsmAnd/src/net/osmand/plus/mapcontextmenu/MapContextMenuFragment.java#L2159)